model/src/main/proto/profile.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.app.model";
+option java_multiple_files = true;
+
+// Top level proto used to store last used id and all profiles
+message Profiles {
```

Comment:
Or perhaps: ``ProfileDatabase``. Not a great name, but I've been thinking of calling the top-level topic cache ``TopicDatabase`` or similar. Eventually this will probably be sqlite, anyway.